doc/api/child_process.md

```diff
@@ -1389,9 +1389,9 @@ connection to the child.
 
 ### `subprocess.signalCode`
 
-* {integer}
+* {string}
```

Comment:
Maybe

```suggestion
* {string|null}
```

considering the description below?

Reply:
While that is the actual type, and correctly so in the typescript typings I also provided a PR for, the type I used for reference in the docs (the close-event) [did not use that |null type convention](https://github.com/nodejs/node/blob/73bc8873598f8ccb9bb768692f5d3e8fa44ba83f/doc/api/child_process.md#event-exit).

It seems to be inconsistent though, as, [in fs there's, in the same createWriteStream options](https://nodejs.org/api/fs.html#fs_fs_createwritestream_path_options) type, both
{Object | null} **Default:** null  
and  
{integer} **Default:** null

[And even within child_process class](https://github.com/nodejs/node/blob/73bc8873598f8ccb9bb768692f5d3e8fa44ba83f/doc/api/child_process.md#child_processspawnsynccommand-args-options) are there examples of this notation.

I don't know what to do here... I don't do well with ambigious choices. I think I prefer the more explicit typing of this suggestion actually, but maybe this should be an overhaul at some point to make this more consistent across docs in general?

Reply:
(I saw you get a thumbs up while typing that last bit where I started "I don't know what to do here" and that just made me realize I should apply this suggestion and move on... Sorry for the ramble)